pkg/dockerfile/generator.go

```diff
@@ -426,9 +426,14 @@ func (g *Generator) installCog() (string, error) {
 	if err != nil {
 		return "", err
 	}
+	pipInstallLine := "RUN --mount=type=cache,target=/root/.cache/pip pip install --no-cache-dir "
+	if !g.IsUsingCogBaseImage() {
+		pipInstallLine += "-t /dep "
+	}
+	pipInstallLine += containerPath
```

Comment:
For consistency, it would be nicer if we added a prefixing space instead of a suffix. That makes it easier to rearrange lines in the future.

```suggestion
	pipInstallLine := "RUN --mount=type=cache,target=/root/.cache/pip pip install --no-cache-dir"
	if !g.IsUsingCogBaseImage() {
		pipInstallLine += " -t /dep"
	}
	pipInstallLine += " " + containerPath
```